### PR TITLE
Portal Browser Demo: Updated WinDesktop and Added Universal projects to use Quartz.

### PR DESCRIPTION
**Pull request by jNery**
@dotMorten : Please code-review, notes logged in the other repo. Thanks.

### CODE REVIEW

File: src/SimplePortalBrowser/PortalBrowser.Universal/App.xaml.cs [hunk 1]
```diff
@@ -29,21 +28,26 @@ sealed partial class App : Application
         /// </summary>
         public App()
         {
-			//TODO: Set your app client id
-			//Esri.ArcGISRuntime.ArcGISRuntimeEnvironment.ClientId = "YOUR CLIENT ID GOES HERE";
-			Esri.ArcGISRuntime.ArcGISRuntimeEnvironment.Initialize();
+            Microsoft.ApplicationInsights.WindowsAppInitializer.InitializeAsync(
+                Microsoft.ApplicationInsights.WindowsCollectors.Metadata |
+                Microsoft.ApplicationInsights.WindowsCollectors.Session);
```
Comment (dotMorten) [inline src/SimplePortalBrowser/PortalBrowser.Universal/App.xaml.cs]: Remove insights

File: src/SimplePortalBrowser/PortalBrowser.Universal/ApplicationInsights.config [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (dotMorten) [inline src/SimplePortalBrowser/PortalBrowser.Universal/ApplicationInsights.config]: delete file

File: src/SimplePortalBrowser/PortalBrowser.Universal/MapPage.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class MapPage : Page
+    {
+        public MapPage()
+        {
+            this.InitializeComponent();
+            MyMapView.LocationDisplay.Start();
```
Comment (dotMorten) [inline src/SimplePortalBrowser/PortalBrowser.Universal/MapPage.xaml.cs]: Wait with OnNavigatedTo before doing this.

**Comment by dotMorten** [comment]
Delete project.json.lock (and add to ignore file)

**Comment by jNery** [comment]
@dotMorten : Updated PR with review feedback.

**Pull request merged**